restic/restic.go

```diff
@@ -106,3 +107,11 @@ type Stats interface {
 	GetJson() []byte
 	GetProm()
 }
+
+func getResticBin() string {
+	resticBin := os.Getenv(resticLocationEnv)
+	if resticBin == "" {
+		resticBin = "/usr/local/bin/restic"
```

Comment:
Shouldn't this be just `restic` and the `$PATH` can handle the rest? Or you can use `LookPath()`: https://golang.org/pkg/os/exec/#LookPath

Reply:
Right, `restic` should be enough here, even for the Docker container.